examples/end-to-end-examples/support_chatbot/mcli_yamls/convert_checkpoint_to_huggingface.yaml

```diff
@@ -0,0 +1,30 @@
+name: convert_to_hf
+
+scheduling:
+  priority: medium
+
+compute:
+  gpus: 0  # Number of GPUs to use
```

Comment:
did this work? i think the way our current system works you still need to request gpus or you don't get any ram/cpu

Reply:
Worked for me on r7z17!